SceneLoader/GLTFVisitor_Material.cpp

```diff
@@ -1,7 +1,7 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
+// Licensed to the .NET Foundation under one or more agreements.

```

Comment:
Are all these file changes necessary?

Reply:
Yes we had to run a script that correctly added all the headers because there was something wrong with my line endings. It was needed to pass the azure pipeline check